boa/src/builtins/regexp/tests.rs

```diff
@@ -98,3 +98,13 @@ fn to_string() {
     );
     assert_eq!(forward(&mut context, "/\\n/g.toString()"), "\"/\\n/g\"");
 }
+
+#[test]
+fn no_panic_on_invalid_character_escape() {
+    let mut context = Context::new();
+
+    assert_eq!(
```

Comment:
I would just try to run it, without checking that it creates a Syntax error, since that will change in the future once we fix regress. If it panics, the test will fail, if it doesn't, it will pass.